Introduction/Getting started/lesson-info.yaml

```diff
@@ -4,5 +4,5 @@ content:
 - Course View
 - Editor
 - Task Description
-- Installation
 - Hello World
+type: Stepik
```

Comment:
I believe it is somehow auto-generated and we probably should remove it.

Reply:
This branch somehow became protected, I can't write to it.